README.md

```diff
@@ -27,22 +27,10 @@ The full list of known Chromedriver versions and their corresponding supported
 Chrome version could be found in
 [mapping.json](https://github.com/appium/appium-chromedriver/blob/master/config/mapping.json)
 
-To download a custom version of Chromedriver, use either npm config property `chromedriver_version`.
+To download a custom version of Chromedriver, please set `CHROMEDRIVER_VERSION` environment variable:
 
 ```bash
-npm install appium-chromedriver --chromedriver_version="2.16"
-```
-
-The property could also be added into your [`.npmrc`](https://docs.npmjs.com/files/npmrc) file.
-
-```bash
-chromedriver_version=2.16
-```
-
-Another possibility is to set the `CHROMEDRIVER_VERSION` environment variable:
-
-```bash
-CHROMEDRIVER_VERSION=2.20 npm install appium-chromedriver
+CHROMEDRIVER_VERSION=107.0.5304.62 npm install appium-chromedriver
```

Comment:
Does this command make sense in general? We usually install UIA2 or Espresso driver via CLI rather than chromedriver itself

Reply:
Yea, usually we expect installing this via drivers like uia2 and espresso. Not sure if we still guide this way, but I just find this non-environment variable usage in Appium org, so wondered removing these usage to reduce confusion